generators/app/index.js

```diff
@@ -77,7 +77,11 @@ module.exports = yeoman.generators.Base.extend({
           }
         },
         validate(name) {
-          return name.includes('-');
+          let nameContainsHyphen = name.includes('-');
+          if (!nameContainsHyphen) {
+            _this.log('\nUh oh, custom elements must include a hyphen in their name. Please try again.');
```

Comment:
doh! Just for my info, how hard is this followed on the team? I've seen a few lines run over by a few chars so I wasn't sure where we stood